test/parallel/test-structuredClone-global.js

```diff
@@ -0,0 +1,18 @@
+// Flags: --expose-internals
+'use strict';
+/* eslint-disable no-global-assign */
+
+require('../common');
+
+const { structuredClone: _structuredClone } = require('internal/structuredClone');
+const {
+  strictEqual
+} = require('assert');
+
```

Comment:
```suggestion

const perf_hooks = require('perf_hooks');

```
The `perf_hooks` subsystem is used but not defined later on.